mne/datasets/sleep_physionet/age.py

```diff
@@ -20,9 +20,8 @@ def fetch_data(subjects, recording=(1, 2), path=None, force_update=False,
                update_path=None, base_url=BASE_URL, on_missing='raise',
                verbose=None):  # noqa: D301
     """Get paths to local copies of PhysioNet Polysomnography dataset files.
-
```

Comment:
docstring error:
```
WARNING: [numpydoc] Validation warnings while processing docstring for 'mne.datasets.sleep_physionet.age.fetch_data':
  SS06: Summary should fit in a single line
```

The blank line here is important, it needs to be put back.  Same error, different test:
https://dev.azure.com/mne-tools/mne-python/_build/results?buildId=12350&view=logs&j=3e7b9abc-cf32-5e9b-ccb3-9355b11c347c&t=9989323c-d330-5eb0-bad7-a6f163ea363a&l=16